packages/blockchain-wallet-v4-frontend/src/modals/RequestCrypto/CoinSelect/index.tsx

```diff
@@ -82,6 +82,9 @@ class RequestCoinSelect extends React.PureComponent<Props> {
               if (account.type === SwapBaseCounterTypes.CUSTODIAL && !data.isAtLeastTier1) {
                 setStep(RequestSteps.IDV_INTRO)
               } else {
+                if (account.type === SwapBaseCounterTypes.CUSTODIAL) {
+                  this.props.requestActions.getIsUserFinishedKyc()
```

Comment:
This will not wait for the result of verification. What is the desired behaviour here?